.github/workflows/ci.yml

```diff
@@ -149,6 +153,18 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # v2.4.2
 
+      - name: Validate build settings
+        run: |
+          VALID="pgsql mysql57 mysql80 sqlite3"
+          DB=${{ matrix.db  }}
```

Comment:
This is the wrong place to validate the matrix, if we're doing matrix validation it should be done in https://github.com/silverstripe/gha-generate-matrix, probably here https://github.com/silverstripe/gha-generate-matrix/blob/1.7/job_creator.php#L587 - we can throw an InvalidArgumentException like we do here https://github.com/silverstripe/gha-generate-matrix/blob/1.7/job_creator.php#L568

There's a list of valid databases here https://github.com/silverstripe/gha-generate-matrix/blob/1.7/consts.php#L36

We'd want a unit test added to https://github.com/silverstripe/gha-generate-matrix/blob/1.7/tests/JobCreatorTest.php

Reply:
I've removed it from this PR. I'll have a glance at adding it at the propose location, but it's not strictly needed for this PR.